theano/configdefaults.py

```diff
@@ -1241,7 +1241,9 @@ def is_valid_check_preallocated_output_param(param):
 def check_mkl_openmp():
     if not theano.config.blas.check_openmp:
         return
-    import os
+    import os, sys
```

Comment:
why the import was here and why you extend it? Both of those import are already at the top of this file.

Reply:
I don't remember why it was there, but I extended it to be sure.